spec/factories/users.rb

```diff
@@ -187,5 +187,14 @@
       confirmation_token { 'token' }
       password { nil }
     end
+
+    trait :proofed do
+      signed_up
+
+      after :build do |user|
+        user.save
+        create(:profile, :active, :verified, :with_pii, user: user)
```

Comment:
the `create` with a user should force the `user` to get saved if it's not (since it needs an ID).

I gave this a shot locally and it still passed:

```suggestion
        create(:profile, :active, :verified, :with_pii, user: user)
```